src/Workspaces/CSharpTest/EmbeddedLanguages/VirtualChars/CSharpVirtualCharServiceTests.cs

```diff
@@ -164,11 +164,7 @@ public void TestEscapeInInterpolatedSimpleString()
 
         [Fact]
         public void TestEscapeInInterpolatedSimpleString_E()
-            => TestFailure("$\"\\e\"");
-
-        [Fact]
-        public void TestEscapeInInterpolatedSimpleString_E_Preview()
-            => Test("$\"\\e\"", @"['\u001B',[2,4]]", CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview));
+            => Test("$\"\\e\"", @"['\u001B',[2,4]]");
```

Comment:
It feels reasonable to update these tests by using LanguageVersion.CSharp12 for the failure case and CSharp13 for the success case, and renaming the tests accordingly